From rdunlap@xenotime.net  Fri Jul 10 10:30:14 2009
From: vibi sreenivasan <vibi_sreenivasan@cms.com>
Date: Wed, 8 Jul 2009 15:37:03 -0700
Subject: driver model: fix show/store prototypes in doc.
Cc: gregkh <greg@kroah.com>, vibi sreenivasan <vibi_sreenivasan@cms.com>
Message-ID: <20090708153703.01fef7e0.rdunlap@xenotime.net>


From: vibi sreenivasan <vibi_sreenivasan@cms.com>

FIX prototypes for show & store method in struct driver_attribute

Signed-off-by: vibi sreenivasan <vibi_sreenivasan@cms.com>
Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/driver-model/driver.txt |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/Documentation/driver-model/driver.txt
+++ b/Documentation/driver-model/driver.txt
@@ -207,8 +207,8 @@ Attributes
 ~~~~~~~~~~
 struct driver_attribute {
         struct attribute        attr;
-        ssize_t (*show)(struct device_driver *, char * buf, size_t count, loff_t off);
-        ssize_t (*store)(struct device_driver *, const char * buf, size_t count, loff_t off);
+        ssize_t (*show)(struct device_driver *driver, char *buf);
+        ssize_t (*store)(struct device_driver *, const char * buf, size_t count);
 };
 
 Device drivers can export attributes via their sysfs directories. 
